packages/@okta/vuepress-site/docs/reference/api/iam-roles/index.md

```diff
@@ -0,0 +1,3027 @@
+
+<ApiOperation method="get" url="/api/v1/groups/${groupId}/roles/${roleId}/targets/groups" />
+
+Lists all Group targets for a `USER_ADMIN` or `HELP_DESK_ADMIN` Role assigned to a Group
+
+###### Request parameters
+
+
+| Parameter       | Description                                                    | Param Type   | DataType   | Required |
+| :-------------- | :------------------------------------------------------------- | :----------- | :--------- | :------- |
+| after           | Specifies the pagination cursor for the next page of targets   | Query        | String     | FALSE    |
+| groupId         | `id` of a Group                                                | URL          | String     | TRUE     |
+| limit           | Specifies the number of results for a page (default is 20)     | Query        | Number     | FALSE    |
+| roleId          | `id` of a Role                                                 | URL          | String     | TRUE     |
+
+Treat the page cursor as an opaque value: obtain it through the next link relation. See [Pagination](/docs/reference/api-overview/#pagination).
+
+###### Response parameters
+
+
+Array of [Groups](/docs/reference/api/groups/)
+
+If the Role isn't scoped to specific Group targets, an empty array `[]` is returned.
+
+###### Request example
+
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/groups/00gsr2IepS8YhHRFf0g3/roles/JBCUYUC7IRCVGS27IFCE2SKO/targets/groups"
+```
+
+###### Response example
+
+
+```json
+[
+    {
+        "id": "00gsrc96agspOaiP40g3",
+        "created": "2019-02-27T15:19:11.000Z",
+        "lastUpdated": "2019-02-27T15:19:11.000Z",
+        "lastMembershipUpdated": "2019-02-27T15:19:11.000Z",
+        "objectClass": [
+            "okta:user_group"
+        ],
+        "type": "OKTA_GROUP",
+        "profile": {
+            "name": "userGroup0",
+            "description": null
+        },
+        "_links": {
+            "logo": [
+                {
+                    "name": "medium",
+                    "href": "http://${yourOktaDomain}/assets/img/logos/groups/okta-medium.d7fb831bc4e7e1a5d8bd35dfaf405d9e.png",
+                    "type": "image/png"
+                },
+                {
+                    "name": "large",
+                    "href": "http://${yourOktaDomain}/assets/img/logos/groups/okta-large.511fcb0de9da185b52589cb14d581c2c.png",
+                    "type": "image/png"
+                }
+            ],
+            "users": {
+                "href": "http://${yourOktaDomain}/api/v1/groups/00gsrc96agspOaiP40g3/users"
+            },
+            "apps": {
+                "href": "http://${yourOktaDomain}/api/v1/groups/00gsrc96agspOaiP40g3/apps"
+            }
+        }
+    }
+]
+```
+
+#### Add a group target to a group administrator role
+
+##### Add a group target to a group administrator role given to a user
```

Comment:
group target to a group administrator role given to a user > Group target to a Group administrator Role given to a User